preupg/settings.py

```diff
@@ -92,27 +89,21 @@
 # name of the file which contains a list of rules
 file_list_rules = "list_rules"
 
-# path to file with definitions of common scripts
-post_script = os.path.join(common_dir, "post_scripts.txt")
-
 # kickstart and postupgrade.d directories
-preupgrade_dirs = [dirty_conf_dir, clean_conf_dir,
-                   'hooks', 'kickstart', postupgrade_dir, 'common',
+preupgrade_dirs = [dirty_conf_dir, clean_conf_dir, 'hooks', 'data/templates',
+                   "data/" + postupgrade_dir, 'data/preassesment',
```

Comment:
This is totaly wrong. !!!! These directories are created by preupg binary before an assessment in /root/preupgrade/ directory. The kickstart directory is used by preupg-kickstart-generated file and contains several files for The Kickstart file. It has an affect to modules. This will cause a several problems. Common is obsolete definitelly.